doc/actions.md

```diff
@@ -507,27 +507,32 @@ MoveResize
 **MoveHorizontal (int)**
 
 Moves a frame horizontally. Int is amount of pixels and can be
-negative.
+negative. Int can be followed by '%' which denotes the percent of the
+screen width.
```

Comment:
yes, docs are updated here too. You think a guy who reported doc problems forgot about doc updates (kidding).

Reply:
Hah! I tricked myself with the review tool. I thought it was rather strange it wasn't included given you've noticed a few doc issues already.